Apps/JobScheduler/src/Jobs/CloseAccountJob.cs

```diff
@@ -80,7 +80,7 @@ public class CloseAccountJob
         [DisableConcurrentExecution(ConcurrencyTimeout)]
         public void Process()
         {
-            DateTime deleteDate = DateTime.Now.Date.AddDays(this.daysBeforeDeletion);
+            DateTime deleteDate = DateTime.UtcNow.Date.AddDays(this.daysBeforeDeletion);
```

Comment:
Is the daysBeforeDeletion variable comming from configuration? if so I think it should be in hours.

Reply:
I've updated this to hours